### ISSUE TITLE
Horizontal bar charts?

### ISSUE THREAD

**Issue by spyhunter99** [timmolter/XChart#136]
Is there a way to swap the axises? Something like this?
![image_total_message_sizes_by_service](https://cloud.githubusercontent.com/assets/1975321/12834894/453c65e2-cb79-11e5-8e06-95a08696c55d.png)

**Comment by timmolter** [comment]
No, unfortunately not yet. I'll leave this issue open as a feature request.

**Comment by on247** [comment]
Can you  offer a bounty on this one? 200 mBtc ?

**Comment by sylvainfurt** [comment]
@timmolter I have started to use XChart and I am interested in making this enhancement if nobody else is already doing it.

I would prefer to not create a generic axis rotator but a new chart type or a new feature of the bar chart to make it horizontal as shown on the screenshot above. 

Thoughts? Guidelines?

**Comment by mogilka** [comment]
Hello! Now the xChart version is 3.6.5. Is there a method that allows to make a horizontal bars in it?

**Comment by timmolter** [comment]
nope, still not implemented.

**Comment by MSWS** [comment]
I would really like for this to be implemented if possible.

**Comment by TonyK59** [comment]
We have 2 charts that have to be horizontal bar charts. I see the request is > 5 years old. Is there a work around even?

**Comment by kimutaku001** [comment]
> l

me 2

**Comment by kimutaku001** [comment]
> We have 2 charts that have to be horizontal bar charts. I see the request is > 5 years old. Is there a work around even?
Now I'm having this problem, and I'm hoping to implement this feature as soon as possible.^_^

**Comment by mgm-rwagner** [comment]
May I ask what the status of this issue is? Will it be resolved at some point?

**Comment by timmolter** [comment]
Implemented via PR #866 which added horizontal bar chart support. Available in the next release.
